content/basics.md

```diff
@@ -369,6 +369,27 @@ Good references:
 - ["On commit messages"](https://who-t.blogspot.com/2009/12/on-commit-messages.html)
 - ["How to Write a Git Commit Message"](https://chris.beams.io/posts/git-commit/)
 
+```{note}
+A great way to learn how to write commit messages and to get inspired by their
+style choices: **browse repositories of codes that you use/like**:
+
+Some examples (but there are so many good examples):
+- [SciPy](https://github.com/scipy/scipy/commits/main)
+- [NumPy](https://github.com/numpy/numpy/commits/main)
+- [Pandas](https://github.com/pandas-dev/pandas/commits/main)
+- [Julia](https://github.com/JuliaLang/julia/commits/master)
+- [ggplot2](https://github.com/tidyverse/ggplot2/commits/main),
+  compare with their [release
+  notes](https://github.com/tidyverse/ggplot2/releases)
+- [Flask](https://github.com/pallets/flask/commits/main),
+  compare with their [release
+  notes](https://github.com/pallets/flask/blob/main/CHANGES.rst)
+
+When designing commit message styles consider also these:
+- How will you easily generate a changelog or release notes?
+- During code review, you can help each other improving commit messages.
+```
+
```

Comment:
```suggestion

But remember: it is better to make any commit, than no commit.  Especially in small projects, weight the benefit with the time it takes.

```

I'm not sure if this is the perfect text or the right place, but I think it's worth emphasizing that it's better to do anything, rather than make it perfect (something not mentoned enough, I think).

Hm, what about a hierarchy of "making commit messages step by step as your project grows": minimal quick thing, slightly better, then Numpy level, for example?
(not needed now, this could be said instead of written)